openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -234,7 +234,7 @@ class RunningScript
   @@run_thread = nil
   @@breakpoints = {}
   @@line_delay = 0.1
-  @@max_output_characters = 8000
+  @@max_output_characters = 50000
```

Comment:
Thought this needed to be bumped up?

Reply:
I couldn't get full stack traces in ScriptRunner.